dot/types/parachain.go

```diff
@@ -0,0 +1,428 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package types
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/crypto"
+	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+)
+
+// BlockNumber represents a block number
+type BlockNumber uint32
```

Comment:
Can we move it to `dot/types/block.go` ?

Reply:
done